drakrun/drakrun/vm.py

```diff
@@ -91,10 +91,31 @@ def is_running(self) -> bool:
         )
         return res.returncode == 0
 
+    """
+    I have a few suggestions that can help remove the raw xl commands from draksetup
+    Here is the abstract, should i pursue with the idea?
+    """
+    # def uptime(self):
+    #   xl uptime command
+
+    # def create(self):
+    #   xl create command
+
+    # def pause(self):
+    #   xl pause command
+
+    # def unpause(self):
+    #   xl unpause command
+
+    # def save(self, pause=False):
+    #   xl save command
```

Comment:
@chivay @icedevml if you can just tell me if these are fine, then I can start working on these

Reply:
:+1: from me